src/page/unsupported.ejs

```diff
@@ -130,24 +134,25 @@
                 </header>
                 <main style="flex: 1 1 0; padding-left: 8px; width: 100%;">
                     <div class="textContent">
-                        <h2 data-uie-name="element-unsupported-headline">This browser is not supported.</h2>
+                        <h2 data-uie-name="element-unsupported-headline">Wire doesn't support this browser.</h2>
```

Comment:
these should all be new translation strings

Reply:
There is a separate ticket for localization of the strings in the "Unsupported browser" page, we agreed with Tom and Sascha to implement it hardcoded EN for now and think about configuring the link URL and translation later.
Since this one should be the simplest plain HTML that can be shown in any oldest browser possible and it has no scripts at all, it might be really difficult to implement the usage of `intl` here.

Reply:
https://wearezeta.atlassian.net/browse/WPB-9005 - the translation ticket